lockapi.pm

```diff
@@ -73,7 +73,7 @@ sub _log {
         $autotest::current_test->remove_last_result;
         $subject .= ' ' . int($args{amend} / ONE_MINUTE) . 'm' . $args{amend} % ONE_MINUTE . 's';
     }
-    testapi::record_info $subject, $msg;
+    record_info $subject, $msg;
```

Comment:
You will need to import that function if you call it without package name. However, I'm not sure how this is simplifying things.

Reply:
Yeah, this is just a ridiculous commit. At the time I am sure it had more meat :)